daikon/pom.xml

```
@@ -233,7 +233,7 @@
                                 <Embed-Dependency>*;artifactId=paranamer|snappy-java|commons-compress|xz;inline=true
                                 </Embed-Dependency>
                                 <Import-Package>
-                                    com.cedarsoftware.*;resolution:=optional,com.fasterxml.*;resolution:=optional,org.apache.*;resolution:=optional,*
+                                    !org.objectweb.asm;!com.github.luben.zstd.*;!org.tukaani.xz.*;!org.brotli.*;com.cedarsoftware.*;resolution:=optional,com.fasterxml.*;resolution:=optional,org.apache.*;resolution:=optional,*
```

```suggestion
                                    org.objectweb.asm;resolution:=optional,com.github.luben.zstd.*;resolution:=optional,org.tukaani.xz.*;resolution:=optional,org.brotli.*;resolution:=optional,com.cedarsoftware.*;resolution:=optional,com.fasterxml.*;resolution:=optional,org.apache.*;resolution:=optional,*
```
I would suggest you make those import optional